compute_endpoint/globus_compute_endpoint/endpoint/interchange.py

```diff
@@ -317,7 +317,7 @@ def process_stored_results():
                         # important to check every iteration as well, so as not to
                         # potentially hang up the shutdown procedure
                         return
-            log.debug("Exit process-stored-results thread.")
```

Comment:
are we losing context if we trim down this log?

Reply:
No; thanks to our logging configuration, this is just being less redundant:

```
... DEBUG ... Pending Task Handler... process_pending_tasks Thread exit
... DEBUG ... Stored Result Handler... process_stored_results Thread exit
... DEBUG ... Pending Result Handler... process_pending_results Thread exit
```